lib/pure/strutils.nim

```diff
@@ -1929,6 +1929,15 @@ proc formatBiggestFloat*(f: BiggestFloat, format: FloatFormatMode = ffDefault,
       # but nothing else is possible:
       if buf[i] in {'.', ','}: result[i] = decimalsep
       else: result[i] = buf[i]
+    when defined(windows):
```

Comment:
Is that Windows specific though or ``vcc`` specific?

Reply:
I'm note entirely sure, but I think VCC (whoever is providing `<stdio.h>`), and apparently all versions except the latest. Checking for VCC + version probably isn't possible, right?

But if the implementation is already correct, the cheap check `result[^4] == '+'` will simply fail, which is negligible in terms of performance of the whole function.

Reply:
Maybe check for VCC at least?
when defined(vcc)

Reply:
Thanks, I missed that there is `defined(vcc)`, was looking for `vcs` ;). Fixed now.